debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/OracleConnectorConfigTest.java

```diff
@@ -200,4 +200,35 @@ public void testSnapshotLockMode() throws Exception {
         connectorConfig = new OracleConnectorConfig(config);
         assertThat(connectorConfig.getSnapshotLockingMode().usesLocking()).isFalse();
     }
+
+    @Test
+    @FixFor("DBZ-3813")
+    public void testDatabasePortAndRacNodeConfigurations() throws Exception {
```

Comment:
Seems we don't have a test with multiple host:port pairs in `rac.nodes`?

Reply:
It should be equivalent to the following but I'll add an explicit case either way:
```
        // Test rac.nodes using combination of with/without port with database.port
        config = Configuration.create().with(port, "1521").with(racNodes, "1.2.3.4,1.2.3.5:1522").build();
        assertThat(config.validateAndRecord(Collections.singletonList(racNodes), LOGGER::error)).isTrue();

        connectorConfig = new OracleConnectorConfig(config);
        assertThat(connectorConfig.getRacNodes()).hasSize(2);
        assertThat(connectorConfig.getRacNodes()).contains("1.2.3.4:1521", "1.2.3.5:1522");
```